controllers/alerts.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const api = require('../api');
+
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
+	const alert = await api(req).get('/alert');
+	const contype = req.headers['content-type'];
```

Comment:
```suggestion
	const contentType = req.headers['content-type'];
```
we use camel case, shortened variables not necessary